### PR TITLE
fix(admin-tool): getting PHP notices when importing Give core Settings #2999

### LINKED ISSUE #2999: fix(admin-tool): getting PHP notices when importing Give core Settings
## User Story

As a `Admin`, I don't  want to get any PHP notices 

I am getting PHP notices when importing Core setting from JSON file 

## Current Behavior

I currently getting PHP notice when importing core setting from JSON file 

## Expected Behavior
I do not expect any PHP notices

## Steps to Reproduce
1. Setup a Frest Sites with Give Core Version 2.1 
2. Now go to Goto Donation > Tool > Export and export Core Setting
3. Now try to import that Core setting byb going to Donation > Tools > Import > Core Settings and import the Setting from JSON file you will get a PHP notices 

## Visuals/ Debbug Log
<details>
[05-Apr-2018 20:12:23 UTC] PHP Notice:  Undefined index: featured_image_size in /app/public/wp-content/plugins/Give/includes/admin/admin-filters.php on line 142
</details>

## Tasks

- [ ] Add condition check before using that variable 

## Environment

<details>
    <summary>WordPress System Info</summary>
### WordPress Environment ###

Home URL: http://give207.com
Site URL: http://give207.com
WP Version: 4.9.5
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: wp_
Table Prefix Length: 3
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: give207.com
TLS Connection: Connection uses The website calling howsmyssl.com's API has been making many calls and does not have a subscription. See https://subscriptions.howsmyssl.com/signup for how to get one.
TLS Connection: Notice:  Undefined property: stdClass::$rating in /app/public/wp-content/plugins/Give/includes/admin/tools/views/html-admin-page-system-info.php on line 199
Server Info: nginx/1.10.1
PHP Version: 7.0.3
PHP Post Max Size: 1,000 MB
PHP Time Limit: 900
PHP Max Input Vars: 3000
PHP Max Upload Size: 1,000 MB
cURL Version: ❌ 7.38.0, OpenSSL/1.0.1t - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 2.1.0
Give Cache: Enabled
Database Updates: All DB Updates Completed.
Give Cache: Enabled
Give Cache: ✔New Donation✔Donation Receipt✔New Offline Donation✔Offline Donation Instructions✔New User Registration✔User Registration Information✔Email access
Upgraded From: –
Test Mode: Enabled
Currency Code: USD
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: http://give207.com/donation-confirmation/
Failure Page: http://give207.com/donation-failed/
Donation History Page: http://give207.com/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Offline Donation
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: N/A
Donor Email Access: Enabled

### Session Configuration ###

Give Use Sessions: Enabled
Session: Disabled

### Inactive Plugins ###

Braintree For WooCommerce Free: by Payment Plugins, support@paymentplugins.com – 2.6.22
Braintree WooCommerce Addon: by Syed Nazrul Hassan – 1.0.0
BuddyPress: by The BuddyPress Community – 2.9.4
Caldera Forms: by Caldera Labs – 1.6.0
Charitable: by WP Charitable – 1.5.13
Constant Contact Plugin for WordPress: by Katz Web Services, Inc. – 4.1.1
Duplicate Post: by Enrico Battocchi – 3.2.1
Email Cop: by Ashfame – 0.1.1
Give - 2Checkout Gateway: by WordImpress – 1.0.2
Give - Authorize.net Gateway: by WordImpress – 1.3.3
Give - AWeber: by WordImpress – 1.0.2
Give - Braintree Gateway: by WordImpress – 1.1
Give - CCAvenue Gateway: by WordImpress – 1.0
Give - Constant Contact: by WordImpress – 1.2.1
Give - ConvertKit: by WordImpress – 1.0.1
Give - CSV Toolbox: by WordImpress – 1.0
Give - Currency Switcher: by WordImpress – 1.0.3
Give - Development Plugin: by Raftaar1191 – 1.0.0
Give - Display Donors: by WordImpress, LLC – 1.0
Give - Dwolla Gateway: by WordImpress – 1.1.2
Give - Email Reports: by WordImpress – 1.1.1
Give - Fee Recovery: by WordImpress – 1.3.8
Give - Form Countdown: by WordImpress – 1.0
Give - Form Field Manager: by WordImpress – 1.2.8
Give - Gift Aid: by WordImpress – 1.1.1
Give - Google Analytics Donation Tracking: by WordImpress – 1.1.3
Give - MailChimp: by WordImpress – 1.3.7
Give - Manual Donations: by WordImpress – 1.2.2
Give - Paymill Gateway: by WordImpress – 1.1
Give - PayPal Pro Gateway: by WordImpress – 1.1.4
Give - PDF Receipts: by WordImpress – 2.2.4
Give - Recurring Donations: by WordImpress – 1.5.7
Give - Stripe Gateway: by WordImpress – 1.5.2
Give - Tributes: by WordImpress – 1.4.1
Give - Zapier: by WordImpress – 1.1.2
Gravity Forms: by rocketgenius – 2.2.6
Gravity Forms Signature Add-On: by Rocketgenius – 3.4
Loco Translate: by Tim Whitlock – 2.1.2
My Calendar: by Joseph C Dolson – 2.5.17
Polylang: by Frédéric Demarle – 2.3.4
WooCommerce: by Automattic – 3.3.4
WooCommerce PayPal Powered by Braintree Gateway: by WooCommerce – 2.1.1
WP Hide & Security Enhancer: by Nsp Code – 1.4.8.2
WPS Hide Login: by WPServeur, NicolasKulka, tabrisrp – 1.2.6.1
YITH Donations for WooCommerce: by YITHEMES – 1.0.17

### Theme ###

Name: Twenty Seventeen
Version: 1.4
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme

</details>

<details>
    <summary>Operating System</summary>
    <ul>
        <li>Platform: Mac OS X</li>
        <li>Version: 1.13.0</li>
    </ul>
</details>

<details>
    <summary>Browser</summary>
    <ul>
        <li>Name: Chrome</li>
        <li>Version: 64.0</li>
    </ul>
</details>

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR to fix #2999 

## How Has This Been Tested?
Manually tested by importing Core Setting on fresh instances 

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/admin-filters.php [hunk 1]
```diff
@@ -136,7 +136,13 @@ function give_import_core_settings_merge_pages( $json_to_array, $type ) {
 function give_import_core_settings_merge_image_size( $json_to_array, $type ) {
 	if ( 'merge' === $type ) {
 		// Featured image sizes import under Display Options > Post Types > Featured Image Size.
-		if ( 'enabled' === $json_to_array['form_featured_img'] ) {
+		if (
+			! empty( $json_to_array['form_featured_img'] )
+			&&
+			! empty( $json_to_array['featured_image_size'] )
```
Comment (DevinWalker) [inline includes/admin/admin-filters.php]: @raftaar1191 why are these on separate lines?

Comment (raftaar1191) [reply includes/admin/admin-filters.php]: @DevinWalker 

It looks good will review the code

Comment (raftaar1191) [reply includes/admin/admin-filters.php]: cleaning up the formatting @DevinWalker

Comment (raftaar1191) [reply includes/admin/admin-filters.php]: @DevinWalker Code formatting updated

**Pull request merged**